### PR TITLE
Edit name in source code and clean up readme

### LINKED ISSUE #8: Clarify name
There are some references to old names in various locations. These should be edited so there's only one name for the app.

### PR DESCRIPTION
**Pull request by peterhgombos**
Fixes #8  and #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,29 @@
...
 
     python manage.py syncdb --migrate
 
-### Lage nye migreringer etter modellendringer
+### Create new migrations or model changes
 
     python manage.py schemamigration <appname> --auto
 
-### deploy til server
+### deploy to server
```
Comment (sigvef) [inline README.md]: Capital D

File: README.md [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
 
-## Komme i gang
+## Installing
 
-virtualenv memeblast && cd memeblast && source bin/activate
-git clone git@bitbucket.org:sigveseb/memeblast.git && cd memeblast
-cp memeblast/memeblast/settings.py.example memeblast/memeblast/settings.py
-pip install -r requirements_dev.txt
-python manage.py syncdb --migrate
+virtualenv memeblast && cd blast && source bin/activate
```
Comment (sigvef) [inline README.md]: Surround this in `\`\`\`` to get that sweet monospaced code formatting.

Comment (stianjensen) [inline README.md]: Rename the virtualenv to blast?

Comment (stianjensen) [inline README.md]: Isn't cloning before creating the virtualenv cooler, btw? This looks old...

Comment (peterhgombos) [inline README.md]: Might be, I just copied it exactly as it was.



Péter H. Gombos

Comment (stianjensen) [inline README.md]: Okay, never mind the order then – but changing only one of the instances of 'memeblast' will introduce an error.

**Comment by stianjensen** [comment]
This will need an overhaul after the Django-1.7 update

**Comment by peterhgombos** [comment]
Updated.

**Comment by stianjensen** [comment]
The migrations you're adding in the migrations folder seem to be remains from when south was used in earlier django versions.

**Comment by peterhgombos** [comment]
Now.

**Pull request merged**